Equipment Inspector failing

**Describe the bug**
`Right Click Player` -> `Gear ...` menu entry stops updating the `Equipment Inspector` panel.

**To Reproduce**
(i am not positive exactly how i triggered the problem)
1. Open RuneLite+ client
2. Enable `Equipment Inspector` plugin
3. Right click player, click `Gear` menu entry
4. Verify `Equipment Inspector` panel has updated successfully
5. Disable `Equipment Inspector` plugin
6. Right click player, click `Gear` menu entry
7. Verify `Equipment Inspector` panel has failed to update

**Expected behavior**
I expected the panel to update correctly both times.

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: 1.5.29-SNAPSHOT
 - Launcher version: 1.6.3-SNAPSHOT

**Additional context**
```
2019-07-07 04:35:35 [pool-2-thread-1] WARN  n.r.c.util.RunnableExceptionLogger - Uncaught exception in runnable net.runelite.client.plugins.equipmentinspector.EquipmentInspectorPlugin$$Lambda$1338/265903559@2c9b8872
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at net.runelite.client.plugins.equipmentinspector.EquipmentInspectorPlugin.lambda$onPlayerMenuOptionClicked$2(EquipmentInspectorPlugin.java:169)
	at net.runelite.client.util.RunnableExceptionLogger.run(RunnableExceptionLogger.java:41)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.reflect.InvocationTargetException: null
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1321)
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1296)
	at javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1348)
	at net.runelite.client.plugins.equipmentinspector.EquipmentInspectorPlugin.lambda$onPlayerMenuOptionClicked$2(EquipmentInspectorPlugin.java:158)
	... 8 common frames omitted
Caused by: java.lang.NullPointerException: null
	at net.runelite.client.plugins.equipmentinspector.EquipmentInspectorPlugin.lambda$null$0(EquipmentInspectorPlugin.java:162)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

--- a/runelite-client/src/main/java/net/runelite/client/plugins/equipmentinspector/EquipmentInspectorPlugin.java
+++ b/runelite-client/src/main/java/net/runelite/client/plugins/equipmentinspector/EquipmentInspectorPlugin.java
@@ -137,7 +137,6 @@ public class EquipmentInspectorPlugin extends Plugin
 			.panel(equipmentInspectorPanel)
 			.build();
 
-
 		pluginToolbar.addNavigation(navButton);
 
 	}
@@ -148,6 +147,7 @@ public class EquipmentInspectorPlugin extends Plugin
 		eventBus.unregister(this);
 
 		menuManager.removePlayerMenuItem(INSPECT_EQUIPMENT);
+		pluginToolbar.removeNavigation(navButton);
 	}
 
 	private void addSubscriptions()
@@ -160,8 +160,6 @@ public class EquipmentInspectorPlugin extends Plugin
 	{
 		if (event.getMenuOption().equals(INSPECT_EQUIPMENT))
 		{
-
-
 			executor.execute(() ->
 			{
 				try

